spec/controllers/application_controller_spec.rb

```diff
@@ -0,0 +1,163 @@
+require 'spec_helper'
+
+describe ApplicationController do
+
+
+  it 'loads the homepage' do 
+    get '/'
+    expect(last_response.status).to eq(200)
+    expect(last_response.body).to include("Welcome to Fwitter")
+  end
+
+  it 'loads the signup page' do 
+    get '/signup'
+    expect(last_response.status).to eq(200)
+  end
+
+  it 'signup directs user to twitter index' do 
```

Comment:
Let's force them to do some validations so they when they get to Rails and we should them rails/AR validations they are in love. Test for blank email/user/pass. Test for pass is greater then 6 characters